etcdserver/etcdserverpb/rpc.proto

```diff
@@ -162,10 +162,14 @@ message RangeRequest {
 	VALUE = 4;
   }
 
-  // key is the first key for the range. If range_end is not given, the request only looks up key.
+  // key is the first key for the range.
   bytes key = 1;
   // range_end is the upper bound on the requested range [key, range_end).
-  // If range_end is '\0', the range is all keys >= key.
```

Comment:
the old line 168 is correct, we should kill the new line168 and line 169.

for line 170,

If the `range_end` is one bit larger than the given `key`, then the range requests get the all keys with the prefix `key`